### ISSUE TITLE
Report problems with Bioschemas mark-up tutorial and FAIR-Checker

### ISSUE THREAD

**Issue by PhilReedData** [BioSchemas/specifications#711]
Report of problems following [tutorial at ACM REP '25 in Vancouver](https://philreeddata.github.io/acmrep25/2_bioschemas.html), July 2025

There is an error showing when I follow the [Bioschemas markup tutorial](https://github.com/zbmed-semtec/bioschemas-ghpages-markup-tutorial). When I use the FAIR-Checker, I get 'Required missing properties'.

I can work around the `conformsTo` property by replacing it with `http://purl.org/dc/terms/conformsTo`. However, that still gives me errors for the license part: "No **dct:conformsTo** property found, using https://bioschemas.org/profiles/SemanticTextAnnotation/0.3-DRAFT profile for validation." What is happening here? The original tutorial doesn't show these errors.

<img width="2918" height="1910" alt="Image" src="https://github.com/user-attachments/assets/4ffc9629-6c7f-4c56-b722-a17f079d4a78" />

In the license section, I tried adding extra JSON for text, conformsTo... and I have no idea what to put for mainEntity.

Has FAIR-Checker got stricter since the tutorial was written?

I created [a fork of the lesson](https://github.com/philreeddata/bioschemas-ghpages-markup-tutorial) so I could make adjustments necessary for a smooth tutorial. I've implemented changes that I determined were necessary. (Thanks to Nick and Helena for my test run (at ISMB/ECCB), and to Michael Sparks (on EVERSE project) who I delivered it to as test runs.)

In summary, I have fixed a few problems (error messages that I guess were not present before eg `conformsTo` in Dataset), taken out the Rich Results Test, and I do not talk about the details of the software page markup. This is so I can explain the parts I most understand in the short time, focusing on the easier concept of dataset (one profile) before they can learn about software (two related profiles) another day.  I still have the outstanding error about FAIR-Checker and licenses, I covered that up and said the tool is being updated. Also, I used the [1.0-RELEASE](https://bioschemas.org/profiles/Dataset/1.0-RELEASE) version of the Dataset profile, not [1.1-DRAFT](https://bioschemas.org/profiles/Dataset/1.1-DRAFT) version which has incomplete documentation.

Ginger told me: 
> the issue is that the JSON-schema validation rule for the "conformsTo" for all profiles was written before we had any consensus on how the value for this property should be formatted. Everyone wanted this property to be included, but there were no specifics as to how. Currently, the conformsTo expects some sort of CreativeWork object with an identifier since all schemas are some sort of CreativeWork in their own right.
> 
> This is how the current validation rules expect to see the values:
```
"conformsTo": {"@type": "CreativeWork", "identifier","https://bioschemas.org/profiles/Dataset/1.0-RELEASE"}
```

[Using the FAIR-Checker](https://fair-checker.france-bioinformatique.fr/inspect) after completing my modified lesson still shows the license error ([example URL](https://duckwhistle.github.io/bioschemas-ghpages-markup-tutorial/dataset.html)):

<img width="1284" height="937" alt="Image" src="https://github.com/user-attachments/assets/985faa66-d494-4059-a08f-36b71f3b9cb7" />

Questions:

1. What bug report, if any, needs to be raised with FAIR-Checker, and how?
2. Where else are there out-of-date or incorrect training materials or tools across Bioschemas?
